spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -678,6 +655,202 @@ boolean result = pollableSource.poll(received -> {
+
+For example, continuing on the previous example and to set up the DLQ with Rabbit binder you need to set the following property
+
+[source,text]
+----
+spring.cloud.stream.rabbit.bindings.input.consumer.auto-bind-dlq=true
+----
+
+Keep in mind that in the above property the word `input` corresponds to the name of the input destination binding.
+The word `consumer` states that it is a consumer property and  `auto-bind-dlq` instructs the binder to configure DLQ for `input`
+destination which results in additional Rabbit queue named `input.myGroup.dlq`.
+
+Once configured, all failed messages will be routed to this queue with error message looking similar to this:
+
+[source,text]
+----
+delivery_mode:	1
+headers:
+x-death:
+count:	1
+reason:	rejected
+queue:	input.hello
+time:	1522328151
+exchange:
+routing-keys:	input.myGroup
+Payload {"name”:"Bob"}
+----
+
+As you can see from the above, your original message is preserved for further actions.
+
+However one thing you may have noticed is that there is limited information on what was the original issue with the message processing. For example, you don’t see a stack
+trace corresponding to the original error.
+To get more relevant information about the original error you must set additional property:
+
+[source,text]
+----
+spring.cloud.stream.rabbit.bindings.input.consumer.republish-to-dlq=true
+----
+
+This will force the internal error handler to intercept the error message and add additional information to it before publishing it to DLQ.
+Once configured you will see the error message contains more information relevant to the original error.
+
+[source,text]
+----
+delivery_mode:	2
+headers:
+x-original-exchange:
+x-exception-message:	has an error
+x-original-routingKey:	input.myGroup
+x-exception-stacktrace:	org.springframework.messaging.MessageHandlingException: nested exception is
+      org.springframework.messaging.MessagingException: has an error, failedMessage=GenericMessage [payload=byte[15],
+      headers={amqp_receivedDeliveryMode=NON_PERSISTENT, amqp_receivedRoutingKey=input.hello, amqp_deliveryTag=1,
+      deliveryAttempt=3, amqp_consumerQueue=input.hello, amqp_redelivered=false, id=a15231e6-3f80-677b-5ad7-d4b1e61e486e,
+      amqp_consumerTag=amq.ctag-skBFapilvtZhDsn0k3ZmQg, contentType=application/json, timestamp=1522327846136}]
+      at org.spring...integ...han...MethodInvokingMessageProcessor.processMessage(MethodInvokingMessageProcessor.java:107)
+      at. . . . .
+Payload {"name”:"Bob"}
+----
+
+====== Re-queue
+
+As mentioned earlier, the currently supported binders (Rabbit and Kafka) rely on `RetryTemplate` to facilitate successful message processing. See <<Retry Template>> for details.
+However, for cases when `max-attempts` property is set to 1, internal reprocessing of the message is disabled, at this point you can facilitate message re-processing (re-tries)
```

Comment:
The comma after "disabled" should be a period, and "at this" should be "At this" (In other words, make two sentences there)
Add a comma after "point"